omod/src/main/resources/config.xml

```diff
@@ -512,6 +512,14 @@
 		</description>
 	</globalProperty>
 
+	<globalProperty>
+		<property>eptsreports.treatmentFollowUpUuid</property>
+		<defaultValue>93603742-1cae-4970-9077-e2b27e46bd7e</defaultValue>
+		<description>
+			Seguimento de tratamento TPT
+		</description>
+	</globalProperty>
+
```

Comment:
This is not formatted and also please follow the same naming of all other config